components/AvalancheCenterLogo.tsx

```diff
@@ -94,48 +96,48 @@ export const AvalancheCenterLogo: React.FunctionComponent<AvalancheCenterLogoPro
   } else {
     actualStyle.height = undefined;
   }
-  actualStyle.aspectRatio = sizes[avalancheCenterId].width / sizes[avalancheCenterId].height;
+  actualStyle.aspectRatio = source[avalancheCenterId].width / source[avalancheCenterId].height;
   return images[avalancheCenterId](actualStyle);
 };
 
-export const preloadAvalancheCenterLogo = async (avalancheCenter: AvalancheCenterID) => {
+export const preloadAvalancheCenterLogo = async (queryClient: QueryClient, avalancheCenter: AvalancheCenterID) => {
   /* eslint-disable @typescript-eslint/no-var-requires */
   switch (avalancheCenter) {
     case 'BTAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/BTAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/BTAC.png')).uri);
```

Comment:
Similar question here - what's the point of pre-fetching a thing that's already in the app bundle? (I feel like I've asked this in Slack and not really understood the answer, sorry that I keep repeating myself)

Reply:
Same answer :)